src/core/processing/qgsprocessingutils.cpp

```diff
@@ -639,9 +639,7 @@ QgsRectangle QgsProcessingUtils::combineLayerExtents( const QList<QgsMapLayer *>
     if ( crs.isValid() )
     {
       //transform layer extent to target CRS
-      Q_NOWARN_DEPRECATED_PUSH
-      QgsCoordinateTransform ct( layer->crs(), crs );
-      Q_NOWARN_DEPRECATED_POP
+      QgsCoordinateTransform ct( layer->crs(), crs, QgsProject::instance()->transformContext() );
```

Comment:
Avoid QgsProject::instance here -- instead, add a QgsProcessingContext argument to the function and use the QgsProcessingContext's transformContext instead.